@here/harp-mapview/lib/poi/PoiRenderer.ts

```diff
@@ -396,11 +396,10 @@ function findImageItem(
     // There is a texture missing in the cache, we attempt again, and then error out.
     const missingTextureCount = missingTextureName.get(imageTextureName);
     missingTextureName.set(imageTextureName, missingTextureCount ? missingTextureCount + 1 : 0);
-    if (missingTextureName.get(imageTextureName)! >= SEARCH_CACHE_ATTEMPTS) {
+    if (missingTextureName.get(imageTextureName)! === SEARCH_CACHE_ATTEMPTS) {
```

Comment:
minor: we could do `missingTextureName.get(imageTextureName)` only once.

Reply:
Yes, but I would need to then do the +1 twice. What is preferred?